pkg/reporting/client.go

```diff
@@ -61,3 +61,30 @@ type RolesInterface interface {
 type EnvironmentInterface interface {
 	Get(name string) (*chef.Environment, error)
 }
+
+type ChefAnalyzeClient struct {
+	Cookbooks         CookbookInterface
+	CookbookArtifacts CBAInterface
+	DataBags          DataBagInterface
+	Environments      EnvironmentInterface
+	Nodes             NodesInterface
+	Policies          PolicyInterface
+	PolicyGroups      PolicyGroupInterface
+	Roles             RolesInterface
+	Search            SearchInterface
+}
+
+func GetChefAnalyzeClient(chefClient *chef.Client) *ChefAnalyzeClient {
```

Comment:
Could you change this to  `NewChefAnalyzeClient`? It's more consistent with naming of other struct-creation functions in the project.

Reply:
Yes, done.